### PR TITLE
[KYUUBI #2000][DEPS] Bump Hadoop 3.3.2

### PR DESCRIPTION
**Pull request by deadwind4**
### _Why are the changes needed?_

Hadoop 3.3.2 is out.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by SteNicholas** [comment]
@deadwind4, please run `build/dependcy.sh` on local to check the dependency.

File: dev/dependencyList [hunk 1]
```diff
@@ -107,6 +106,7 @@ simpleclient_tracer_otel/0.14.1//simpleclient_tracer_otel-0.14.1.jar
 simpleclient_tracer_otel_agent/0.14.1//simpleclient_tracer_otel_agent-0.14.1.jar
 slf4j-api/1.7.35//slf4j-api-1.7.35.jar
 snakeyaml/1.28//snakeyaml-1.28.jar
+snappy-java/1.1.8.2//snappy-java-1.1.8.2.jar
```
Comment (pan3793) [inline dev/dependencyList:109]: Seems does not requied, would you please exclude it from hadoop-client-runtime?

**Review by yaooqinn — APPROVED** [review]

**Review by SteNicholas — COMMENTED** [review]
LGTM.

**Review by SteNicholas — APPROVED** [review]
LGTM. @yaooqinn @pan3793, please help to merge to master.

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**